src/main/java/com/google/firebase/remoteconfig/internal/TemplateResponse.java

```diff
@@ -110,27 +96,22 @@ public RemoteConfigParameter toRemoteConfigParameter() {
     @Key("useInAppDefault")
     private Boolean inAppDefaultValue;
 
-    public ParameterValueResponse() {
-    }
-
-    private ParameterValueResponse(String value, Boolean inAppDefaultValue) {
-      this.value = value;
-      this.inAppDefaultValue = inAppDefaultValue;
+    public String getValue() {
+      return value;
     }
 
-    public static ParameterValueResponse ofValue(String value) {
-      return new ParameterValueResponse(value, null);
+    public Boolean isInAppDefaultValue() {
+      return inAppDefaultValue;
     }
 
-    public static ParameterValueResponse ofInAppDefaultValue() {
-      return new ParameterValueResponse(null, true);
+    public ParameterValueResponse setValue(String value) {
+      this.value = value;
+      return this;
     }
 
-    public RemoteConfigParameterValue toRemoteConfigParameterValue() {
-      if (this.inAppDefaultValue != null && this.inAppDefaultValue) {
-        return RemoteConfigParameterValue.inAppDefault();
-      }
-      return RemoteConfigParameterValue.of(this.value);
+    public ParameterValueResponse setInAppDefaultValue(Boolean inAppDefaultValue) {
```

Comment:
Implement a setter that matches the getter you choose to implement.

Reply:
Should the getter take a `boolean`?